docs/src/examples/sensitivity-analysis-ridge.jl

```diff
@@ -64,76 +64,104 @@ function fit_ridge(X, Y, alpha = 0.1)
     set_silent(model)
     @variable(model, w) # angular coefficient
     @variable(model, b) # linear coefficient
-    @variable(model, e[1:N]) # approximation error
-    ## constraint defining approximation error
-    @constraint(model, cons[i=1:N], e[i] == Y[i] - w * X[i] - b)
+    ## expression defining approximation error
+    @expression(model, e[i=1:N], Y[i] - w * X[i] - b)
     ## objective minimizing squared error and ridge penalty
     @objective(
         model,
         Min,
-        dot(e, e) + alpha * (sum(w * w) + sum(b * b)),
+        1 / N * dot(e, e) + alpha * (w^2 + b^2),
     )
     optimize!(model)
-    return model, w, b, cons # return model, variables and constraints references
+    return model, w, b # return model & variables
 end
 
 
-# Train on the data generated.
+# Plot the data points and the fitted line for different alpha values
 
-model, w, b, cons = fit_ridge(X, Y)
-ŵ, b̂ = value(w), value(b)
-
-# We can visualize the approximating line.
-
-p = Plots.scatter(X, Y, label="")
+p = Plots.scatter(X, Y, label=nothing, legend=:topleft)
 mi, ma = minimum(X), maximum(X)
-Plots.plot!(p, [mi, ma], [mi * ŵ + b̂, ma * ŵ + b̂], color=:red, label="")
+Plots.title!("Fitted lines and points")
 
+for alpha in 0.5:0.5:1.5
+    local model, w, b = fit_ridge(X, Y, alpha)
+    ŵ = value(w)
+    b̂ = value(b)
+    Plots.plot!(p, [mi, ma], [mi * ŵ + b̂, ma * ŵ + b̂], label="alpha=$alpha", width=2)
+end
+p
 
 # ## Differentiate
 
 # Now that we've solved the problem, we can compute the sensitivity of optimal
-# values of the angular coefficient `w` with
+# values of the slope `w` with
 # respect to perturbations in the data points (`x`,`y`).
 
-# Begin differentiating the model.
-# analogous to varying θ in the expression:
-# ```math
-# e_i = (y_{i} + \theta_{y_i}) - w (x_{i} + \theta_{x_{i}}) - b
-# ```
+alpha = 0.4
+model, w, b = fit_ridge(X, Y, alpha)
+ŵ = value(w)
+b̂ = value(b)
+
+# We first compute sensitivity of the slope with respect to a perturbation of the independent
+# variable `x`.
```

Comment:
calling x a variable will be a bit confusing

Reply:
yes, at the same time from the statistical point of view independent variable is the appropriate concept here